### ISSUE TITLE
Computing performance of Models built with various activation functions

### ISSUE THREAD

**Issue by jkk2000** [WebClub-NITK/Hacktoberfest-2k20#58]
### Description
Given the Fashion mnist dataset, construct models (a simple model with 4 hidden layers of same activation function would suffice) using different activation functions like ReLU, SeLU, ReLU variants, sigmoid. Plot the accuracy of each model with respect to the test data using matplotlib. *P.S. Use softmax as the output layer*
Dataset can be downloaded for [here](https://www.kaggle.com/zalando-research/fashionmnist)
Code for dataset:
```
from tensorflow import keras
fashion_mnist = keras.datasets.fashion_mnist
(train_images, train_labels), (test_images, test_labels) = fashion_mnist.load_data()
```

### Details
- Technical Specifications: python, numpy, pandas, keras, tensorflow
- Type of issue: Multiple
- Time Limit: 2 Days.

### Issue requirements / progress
Train your model on the data and provide the average accuracy, use vanilla stochastic gradient descent.
**Note: Each PR should be a solution using only one model**
* Model built using sigmoid function
* Model built using ReLU
* Model built using SeLU
* Model built using Tanh
* Model built using Leaky ReLU

### Resources
[Example](https://www.tensorflow.org/tutorials/keras/classification)
[Jupyter Notebook Tutorial](https://www.dataquest.io/blog/jupyter-notebook-tutorial/)
[Pandas Tutorial](https://pandas.pydata.org/pandas-docs/stable/user_guide/10min.html)
[Different Activation Functions](https://keras.io/api/layers/activations/)

### Directory Structure
Within this repository's 'machine learning' folder, create a folder called 'Covid-19 Data Visualization' (if it doesn't exist yet). Within this folder, place the file/folder in which you have written the program for the task, named appropriately.

#### Note
1. Please claim the issue first by commenting here before starting to work on it.
2. Once you are done with the task and have created a Pull Request, please tag @jkk2000 to request a review.

**Comment by KangHyeoni** [comment]
Hello @jkk2000, can I work on it?

**Comment by jkk2000** [comment]
> Hello @jkk2000, can I work on it?

@KangHyeoni Go ahead

**Comment by KangHyeoni** [comment]
@jkk2000, I have a question. Should all models be trained in same number of epoch? Some models have good accuracy with only 5 epochs, while some needs almost 50 epochs.

**Comment by jkk2000** [comment]
@KangHyeoni Yes, train all models for at most 25 epochs, then stop.

**Comment by KangHyeoni** [comment]
@jkk2000 Thank you for answer!
